docs/source/en/api/pipelines/text_to_video_zero.md

```diff
@@ -149,6 +162,34 @@ To generate a video from prompt with additional pose control
     imageio.mimsave("video.mp4", result, fps=4)
     ```
 
+#### SDXL Support
+
+Since our attention processor also works with SDXL, it can be utilized to generate a video from prompt using ControlNet models powered by SDXL:
+
+```python
+import torch
+from diffusers import StableDiffusionXLControlNetPipeline, ControlNetModel
+from diffusers.pipelines.text_to_video_synthesis.pipeline_text_to_video_zero import CrossFrameAttnProcessor
+
+controlnet_model_id = 'thibaud/controlnet-openpose-sdxl-1.0'
+model_id = 'stabilityai/stable-diffusion-xl-base-1.0'
+
+controlnet = ControlNetModel.from_pretrained(controlnet_model_id, torch_dtype=torch.float16)
+pipe = StableDiffusionControlNetPipeline.from_pretrained(
+	model_id, controlnet=controlnet, torch_dtype=torch.float16
+).to('cuda')
+
+# Set the attention processor
+pipe.unet.set_attn_processor(CrossFrameAttnProcessor(batch_size=2))
+pipe.controlnet.set_attn_processor(CrossFrameAttnProcessor(batch_size=2))
+
+# fix latents for all frames
+latents = torch.randn((1, 4, 128, 128), device="cuda", dtype=torch.float16).repeat(len(pose_images), 1, 1, 1)
+
+prompt = "Darth Vader dancing in a desert"
+result = pipe(prompt=[prompt] * len(pose_images), image=pose_images, latents=latents).images
```

Comment:
Where do `pose_images` come from - could we define them here somehow so that the example works?

Reply:
@patrickvonplaten `pose_images` are defined in the same sub-section of Text-To-Video with Pose Control; as only the third step is different in case of SDXL inference, I have added the SDXL support in form of a bullet point to make that more explicit.